GH#27491: Added Network Time Protocol (NPT) as a Networking req

skrthomas opened the pull request:
Git Hub Issue burndown: Issue #27491 
For 4.6+
Added Network Time Protocol (NPT) as a Networking Req. for vsphere installation.
Link to doc preview: https://deploy-preview-30926--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned

bergerhoffer commented on modules/installation-vsphere-installer-infra-requirements.adoc:
```diff
@@ -85,6 +85,10 @@ Available resources vary between clusters. The number of possible clusters withi
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-networking-NTP_{context}"]
+=== Network Time Protocol (NTP)
+Each OpenShift Container Platform node in the cluster must have access to an NTP server discoverable via DHCP.
```
* Openshift Container Platform -> {product-title}
* suggest changing "to an NTP server discoverable via DHCP" -> "to an NTP server that is discoverable via DHCP", for clarity

skrthomas replied:
@bergerhoffer ooo yes, applies to 4.5+ and ok pushed those comments :)

bergerhoffer reviewed (commented):
Just one comment!

Do you know which versions this applies for?

jeana-redhat commented:
Just noticed this is a twin to the issue I picked up, https://github.com/openshift/openshift-docs/issues/27490 (resolved in https://github.com/openshift/openshift-docs/pull/30928). Only mentioning here for crosslinking :)

ryandevlin-redhat commented:
This still applies in 4.7+. One note, maybe this should be reworded as a warning instead of a must have? It is _physically possible_ to install IPI without NTP, but if the servers are too far out of sync it will fail. You can attempt to get the clocks on the servers as close as possible and then attempt an IPI install, but if the clocks drift you can get problems like "bad cert" errors.

So while it's technically not a "_must-have_", IPI can be very tedious without it and I think the docs should highlight that fact.

skrthomas commented:
@ryandevlin-redhat Thanks for your feedback. I updated this to be a note instead. Please let me know if you have any additional feedback.
updated doc preview link: https://deploy-preview-30926--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html

ryandevlin-redhat commented:
The note looks great! Do you think it would be possible to move it directly above the **"Required IP Addresses"** paragraph? That way it is below the paragraph regarding DHCP, which is a more relevant spot for that note.

skrthomas commented:
@ryandevlin-redhat going up! :)

ryandevlin-redhat commented:
@skrthomas Awesome! Lgtm

bergerhoffer commented:
/remove-lifecycle rotten

bergerhoffer commented:
/cherrypick enterprise-4.9

bergerhoffer commented:
/cherrypick enterprise-4.7
